drivers/hud_pit/app/models/hud_pit/generators/pit/fy2024/adult_and_child.rb

```diff
@@ -0,0 +1,88 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudPit::Generators::Pit::Fy2024
+  class AdultAndChild < Base
+    QUESTION_NUMBER = 'Households with at least one Adult & one Child'.freeze
```

Comment:
Probably not worth changing, but the language looks to be `Adult & Child (at least one adult and one child)`